systemuser.sh

```diff
@@ -84,6 +85,14 @@ sed -i "s/python/python$PYVERSION/g" $KERNEL_DIR/root/kernel.json # Set Python v
 # R
 cp -rL $LCG_VIEW/share/jupyter/kernels/ir $KERNEL_DIR
 sed -i "s/IRkernel::main()/options(bitmapType='cairo');IRkernel::main()/g" $KERNEL_DIR/ir/kernel.json # Force cairo for graphics
+# Octave
+OCTAVE_KERNEL_PATH=$LCG_VIEW/share/jupyter/kernels/octave
+if [[ -d $OCTAVE_KERNEL_PATH ]];
+then
+   cp -rL $OCTAVE_KERNEL_PATH $KERNEL_DIR
+   export OCTAVE_KERNEL_JSON=$KERNEL_DIR/octave/kernel.json
```

Comment:
Do we need this variable to be exported?

Reply:
Yes. This is the variable that the kernel picks to find the json file. (this was the PR I did to them)